kokoro/linux/bazel/build.sh

```diff
@@ -7,3 +7,4 @@ cd $(dirname $0)/../../..
 
 git submodule update --init --recursive
 bazel test :protobuf_test
+cat /tmpfs/tmp/bazel/execroot/com_google_protobuf/bazel-out/k8-fastbuild/testlogs/protobuf_test/test.log
```

Comment:
Is this path stable? Can we use a path that doesn't depend on cpu or build type?

Reply:
I will remove it before merge.